graphics/vpype/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vpype
+version             1.9.0
+revision            0
+
+checksums           rmd160  b56bed854d574f69bf3c264d5c08c7061e238f6d \
+                    sha256  1b09fcd67ba911d8f558a09e3da6ae655ed921b7de599ebcacc7090550f77a71 \
+                    size    476328
+
+description         The Swiss-Army-knife command-line tool for plotter vector graphics.
+long_description    ${description}
+
+categories          graphics
```

Comment:
please change this to `categories-prepend` so that `python` will still be part of it